wiki/wikiproject_klara_weißbender.qmd

```diff
@@ -0,0 +1,21 @@
+---
+title: "Wiki Entry Template"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: Klara Weißbender 
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+Key Quote: "The central aims of the present study were...to provide a theoretical and empirical extension of previous research by accounting for the conditional nature of digital stress effects through the introduction of two central situational boundary conditions (i.e., goal conflict and autonomy need dissatisfaction) identified in the mobile connectivity paradox (Vanden Abeele, 2021) and the IM3UNE model (Schneider et al., 2022)"(Gilbert et al., 2022).
+
+Main Argument: The study of Gilbert et al. (2022) shows the importance of situational boundary conditions in regard to the phenomenon of digital stress.The study integrates two recent models -the connectivity paradox by Vanden Abeele and the IM3UNE by Schneider et al.- which both emphasize the effects of digital communication on psychological well-being (Gilbert et al., 2022, p. 6). The study also identifies two situational moderators (online vigilance and autonomy need dissatisfaction) of the effects of online vigilance and smartphone use behavior on stress(Gilbert et al., 2022, p. 5).These two situational moderators are significant within-person moderators and were derived from the previous mentioned models (Gilbert et al., 2022, p. 16).The findings of the study provide us with a larger understanding of the situational dynamics connected to digital stress and have practical implications for digital stress reduction (Gilbert et al., 2022, p. 17-18)
+
+Relationship: A lot of studies use the connectivity paradox from Vanden Abeele (2021) as a baseline for their empirical work. A lot of these studies also derive variables from this paper and essentially say the same thing as the study of Gilbert et al.. Most of the time these papers are about psychological well being and perceived autonomy just like Gilbert et al.. For example the study of Granow et al. (2018) deals with these themes as well. But it just chooses a different starting point and deals with the phenomenen of binge watching. 
+
+Questions: Further research should explore whether online vigilance and autonomy need dissatisfaction are the only two situational boundary conditions or whether there are more. And it should be explored if other situational boundary conditions show bigger interaction effects with the two proposed predictors of stress on the within-person level.Because online vigilance and autonomy need dissatisfaction only show small ones. 
```

Comment:
What could be other relevant boundary conditions? Make some suggestions!

Try to provide some ideas on how to systematise boundary conditions.

Reply:
Other relevant boundary conditions could be: autonomy need satisfaction, competence or relatedness 

Maybe you could systematise the boundary conditions in positive and negative conditions.